port_flatpak.patch

```diff
@@ -0,0 +1,166 @@
+ url = https://help.sugarlabs.org/en/typing_turtle.html
+diff --git a/balloongame.py b/balloongame.py
+index 746b602..4c1558b 100644
+--- a/balloongame.py
++++ b/balloongame.py
+@@ -245,7 +245,7 @@ class BalloonGame(Gtk.VBox):
+     
+         cr.set_source_rgb(0, 0, 0)
+         pango_layout = PangoCairo.create_layout(cr)
+-        fd = Pango.FontDescription('Times')
++        fd = Pango.FontDescription('Monospace')
+         fd.set_size(12 * Pango.SCALE)
+         pango_layout.set_font_description(fd)
+         pango_layout.set_text(report, len(report))
+@@ -354,7 +354,7 @@ class BalloonGame(Gtk.VBox):
+ 
+     def queue_draw_score(self):
+         layout = self.area.create_pango_layout(_('SCORE: %d') % self.score)
+-        layout.set_font_description(Pango.FontDescription('Times 14'))
++        layout.set_font_description(Pango.FontDescription('Monospace 14'))
+         size = layout.get_size()
+         x = self.bounds.width-20-size[0]/Pango.SCALE
+         y = 20
+@@ -363,7 +363,7 @@ class BalloonGame(Gtk.VBox):
+     def draw_score(self, cr):
+         cr.set_source_rgb(0, 0, 0)
+         pango_layout = PangoCairo.create_layout(cr)
+-        fd = Pango.FontDescription('Times')
++        fd = Pango.FontDescription('Monospace')
+         fd.set_size(14 * Pango.SCALE)
+         pango_layout.set_font_description(fd)
+         text = _('SCORE: %d') % self.score
+@@ -380,7 +380,7 @@ class BalloonGame(Gtk.VBox):
+         # Draw instructions.
+         cr.set_source_rgb(0, 0, 0)
+         pango_layout = PangoCairo.create_layout(cr)
+-        pango_layout.set_font_description(Pango.FontDescription('Times 14'))
++        pango_layout.set_font_description(Pango.FontDescription('Monospace 14'))
+         text = _('Type the words to pop the balloons!')
+         pango_layout.set_text(text, len(text))
+         size = pango_layout.get_size()
+diff --git a/editlessonscreen.py b/editlessonscreen.py
+index 8d7e8ec..5c73d5a 100644
+--- a/editlessonscreen.py
++++ b/editlessonscreen.py
+@@ -543,7 +543,7 @@ class EditLessonScreen(Gtk.VBox):
+         # Load the myspell dictionary.
+         # TODO: Find a better way to determine its location.
+         code = locale.getdefaultlocale()[0] or 'en_US'
+-        return lessonbuilder.load_wordlist('/usr/share/myspell/%s.dic' % code)
++        return lessonbuilder.load_wordlist('/app/share/myspell/%s.dic' % code)
+ 
+     def generate_ok_clicked_cb(self, btn, box):
+         self.save()
+diff --git a/keyboard.py b/keyboard.py
+index 48bdbfc..82eb7ee 100644
+--- a/keyboard.py
++++ b/keyboard.py
+@@ -106,14 +106,7 @@ def _is_olpcm_model():
+     keyboard, and 'olpc' for membrane keyboard.
+ 
+     """
+-    code = None
+-    p = subprocess.Popen(["setxkbmap", "-query"], stdout=subprocess.PIPE)
+-    out, err = p.communicate()
+-    
+-    for line in out.decode('utf-8').splitlines():
+-        if line.startswith('model:'):
+-            code = line.split()[1]
+-    return code == 'olpcm'
++    pass
```

Comment:
Why the need to remove this? Shouldn't it just `False`?

Reply:
Fixed on https://github.com/tchx84/flathub/pull/4/commits/c57aae07c2d163c2ca31e2e2eced9a3662063cad